README.md

```diff
@@ -1,11 +1,4 @@
-# Set up a "Hello Microverse" project
```

Comment:
- [ ] Great job on adding a README to your project. but it seems to miss important information, remember the repository has to be professional. and the README is what recruiters see first, that's why we need to keep it professional. You can use [this template](https://github.com/microverseinc/readme-template) and customize it.